R/model.R

```diff
@@ -183,14 +236,28 @@ fall_run_model <- function(scenario = NULL, mode = c("seed", "simulate", "calibr
                                fecundity = ..params$spawn_success_fecundity,
                                stochastic = stochastic)
 
-
-    # # For use in the r2r metrics
+    # R2R hatchery logic -------------------------------------------------------
+    # Currently adds only on major hatchery rivers (American, Battle, Feather, Merced, Moke)
+    # Add all as fry? or should we do larger
+    natural_juveniles <- rowSums(juveniles * natural_proportion_with_renat)
```

Comment:
simple optimization: multiplying each col then adding is the same as adding them all then multiplying

```r
total_juves_pre_hatchery <- rowSums(juveniles)
natural_juveniles <- total_juves_pre_hatchery  * natural_proportion_with_renat
```

Reply:
updated!